docs/cli/edgedb_analyze.rst

```diff
@@ -9,23 +9,31 @@ edgedb analyze
 
     This CLI feature is compatible with EdgeDB server 3.0 and above.
 
+.. note::
+
+    This feature is also available inside the EdgeDB REPL and UI in which
+    prefacing it with ``edgedb`` is not required.
```

Comment:
Here's a suggestion that I believe will incorporate both my notes and @scotttrinh's notes from the previous comment thread.
```suggestion
    Performance analysis is also available in our :ref:`CLI REPL
    <ref_cli_edgedb>` and the UI's REPL and query builder (both accessible by
    running :ref:`ref_cli_edgedb_ui` to invoke your instance's UI). Use it by
    prepending your query with ``analyze``.
```